ui/media/js/utils.js

```diff
@@ -720,9 +720,140 @@ Array.prototype.addEventListener = function(method, callback) {
     const originalFunction = this[method]
     if (originalFunction) {
         this[method] = function() {
-            console.log(`Array.${method}()`, arguments)
```

Comment:
Removed the console.log fromhttps://github.com/cmdr2/stable-diffusion-ui/pull/1087